.babelrc

```diff
@@ -15,13 +15,6 @@
     ]
   ],
   "plugins": [
-    [
```

Comment:
This would cause jest to conflict in a very weird way with some antd components.

It seems to be a well know issue and this change seemed to fix it. Based on https://github.com/facebook/jest/issues/6938#issuecomment-545451157

Reply:
doesn't removing this piece of code break some other stuff as well though?

Reply:
I tried running tests without problems and going through the application locally it seems to be ok.

I'll test the staging once it is up

Reply:
Also I messages @marcellp in case he has more context on possible problems this removal might cause.

If it does cause many problems there is another "solution" which is to extract components from antd objects without object destructuring (which is what fails in the tests).

So this breaks the tests:
`
import b from 'antd';
const {a} = b;
`

This doesn't 
`
import b from 'antd';
const a = b.a;
`

Reply:
interesting, we could just as well be fine without it. Let's try it in staging and see if it is ok

Reply:
Turns out it did break staging, just had a call with marcell and he figured out a way to have this work correctly (the latest commit).